package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "next dev",
-    "build": "next build && sed -i 's/\"type\": \"module\"/\"type\": \"commonjs\"/' package.json",
+    "build": "if [ $(uname) = 'Darwin' ]; then next build && sed -i '' 's/\"type\": \"module\"/\"type\": \"commonjs\"/' package.json; else next build && sed -i 's/\"type\": \"module\"/\"type\": \"commonjs\"/' package.json; fi",
```

Comment:
Reverted in this commit, because it is not important for MacOS
[d783dd7](https://github.com/upleveled/next-starter-peacock/pull/153/commits/d783dd797b72f70a80d8b3195356271bff071456)